rfd/0091-non-ad-desktop-access.md

```diff
@@ -0,0 +1,144 @@
+---
+authors: Przemko Robakowski (przemko.robakowski@goteleport.com)
+state: draft
+---
+
+# RFD 91 - Desktop access to hosts that are not connected to Active Directory
+
+## Required Approvers
+
+- Engineering: @zmb3 && (@ibeckermayer || @LKozlowski)
+- Product: (@klizhentas || @xinding33)
+
+## Overview
+
+Common ask for Desktop Access in Teleport is to connect to hosts that are not part of the domain, so there's no Active
+Directory available and no Kerberos involved. Unfortunately Windows supports smart card login, used currently in
+Teleport, only for domain-joined hosts, so we need another solution for non-AD case.
+
+Idea is to use install on host DLL library containing custom
```

Comment:
I assume (to start at least) that very little of this code overlaps with anything Teleport does today. Does that mean we should create a new repo for this Windows package, or do you plan to include the source in this repository?